muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/workspace/rest/WorkspaceRESTService.java

```diff
@@ -1604,7 +1608,17 @@ public Response getFileAnswer(@PathParam("FILEID") String fileId) {
       }
     }
     
-    return Response.ok(answerFile.getContent())
+    byte[] content = answerFile.getContent();
+    if (content == null) {
+      try {
+        content = fileAnswerUtils.getFileContent(workspaceMaterialReply.getUserEntityId(), answerFile.getFileId());
```

Comment:
getFileContent currently may return byte[0] if not changed - should this be then status(NOT_FOUND)?